src/redux/books/books.js

```diff
@@ -0,0 +1,26 @@
+const ADD = 'my-app/books/ADD';
+const REMOVE = 'my-app/books/REMOVE';
+
+export default function bookReducer(state = [], action) {
+  switch (action.type) {
+    case ADD:
+      return [
+        ...state,
+        {
+          id: action.id,
+          author: action.author,
+          title: action.title,
+        },
+      ];
+    case REMOVE:
+      return state.map((book) => book.id !== action.id);
```

Comment:
- [ ] OPTIONAL: For this kind of functionality, I recommend using filter() instead of map() so you can filter the array of unwanted elements 💪

Reply:
Thank you for the feedback

Reply:
Thank you for the feed back